addons/mail/static/src/components/message/message_tests.js

```diff
@@ -189,7 +189,8 @@ QUnit.test('Notification Sent', async function (assert) {
     assert.expect(9);
 
     await this.start();
-    const threadView = this.env.models['mail.thread_view'].create({
+    const threadViewer = this.env.models['mail.thread_viewer'].create({
+        hasThreadView: true,
```

Comment:
Not super fond of how this field work... that `mail.thread_viewer` must be aware of `chatter` or `chatWindow` or `chatter`.

Not reasonable to have clean solution in this PR though...

Reply:
Indeed, we lack technical expressiveness for this kind of feature.